calculator/src/components/display.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import './css/button.css'
+
+const Display = ({state, handleNumberClick}) => {
+    
+        return(
+         <div className="display">
+         <h2>{state.displayValue}</h2>
+         <div className="AllNumbers">
+            <button className="0" value="0" onClick={(e) =>handleNumberClick(e)}>0</button>
+            <button className="1" value="1" onClick={(e) => handleNumberClick(e)}>1</button>
+            <button className="2" value="2" onClick={(e) => handleNumberClick(e)}>2</button>
+            <button className="3" value="3" onClick={(e) => handleNumberClick(e)}>3</button>
+            <button className="4" value="4" onClick={(e) => handleNumberClick(e)}>4</button>
+            <button className="5" value="5" onClick={(e) => handleNumberClick(e)}>5</button>
+            <button className="6" value="6" onClick={(e) => handleNumberClick(e)}>6</button>
+            <button className="7" value="7" onClick={(e) => handleNumberClick(e)}>7</button>
+            <button className="8" value="8" onClick={(e) => handleNumberClick(e)}>8</button>
+            <button className="9" value="9" onClick={(e) => handleNumberClick(e)}>9</button>
+            <button className="+" value="+" onClick={(e) => handleNumberClick(e)}>+</button>
```

Comment:
This should have its own separate function ex. `handleOperationClick`. The user should not see the operation signs on the screen.